Source/WebKit/UIProcess/Cocoa/GroupActivities/WKGroupSession.swift

```diff
@@ -142,8 +162,10 @@ fileprivate extension WKGroupSessionState {
         }
     }
 
-    deinit {
-        incomingSessionsTask?.cancel()
+    @objc deinit {
+//        Task { @MainActor [incomingSessionsTask] in
```

Comment:
Remove or uncomment. Although I’m fairly certain you don’t need an explicit actor annotation here:
```swift
@objc deinit {
    Task { await incomingSessionsTask?.cancel() }
}
```

Reply:
Or potentially (to avoid capture `self` during its dealloc):
```swift
@objc deinit {
    Task { [incomingSessionsTask] in incomingSessionsTask?.cancel() }
}
```